### ISSUE TITLE
Setup Paypal - Linux 

### ISSUE THREAD

**Issue by Iluminatos** [sant0ro/chocolatey#195]
Hi.

Im currently using the BETA of Chocolatey. And im checking so everything works but when i try to checkout with a package for Paypal it wont work it says this:
[](https://gyazo.com/7c020b371ba419246ec01beb646ca3a6)
Is there a config i might need to setup?

**Comment by sant0ro** [comment]
Hi @Iluminatos!

Thanks for contributing on **Chocolatey** repository.

We're glad that you're using our software. I need to remember that **this** message was generated automatically.

We need clarify some things before we can review & approve your **Pull Request**:

* Remember, only create a **Pull Request** to the `development` branch.
* Explain your changes on your **Pull Request**
* You need specify the purpose of the change
* We also recommend to add screenshots of what was added.
* We also recommend that you inform why those changes are useful.

Recommendations About your recently opened PR?
--------------------------------

We really recommend using [Lightshot](https://app.prntscr.com/pt-br/download.html) to take your screenshot. It allows marks, annotations and more.

Also we recommend specify some Informations about your **OS** like.:

* Operational System
* Which Web Server are you using (like XAMPP)
* Which version of Web Server Using (eg.: 1.7.3)
* Which version of php & mysql & apache using. (Specify if is possible)
* How much RAM does you have?

I created my PR and now?
---------------------------

You wait.

We have **ful-jobs** and a **life**, we will receive updates by e-mail and soon as possible update your **PR** by tagging it correctly, assigning to our team and **obviously** answering it.

We will **review** and if everything **OK** approve your PR with some time. Since we need analyze what's happening here.

Remember to test your changes before commiting it.

Remember also to undo any change on configuration files that you may have done.

Further information
-------------------

* This **BOT** was originally created by @ajdlinux and @daxtens.
* Edited by @sant0ro for own purposes.
* Contributors of Chocolatey @BobyTS and @s150217754

**Comment by sant0ro** [comment]
Hi @Iluminatos!

Thanks for reporting an Issue to the **Chocolatey Repository**.

We're glad that you're using our software. I need to remember that **this** message was generated automatically.

In order to **gracefully** report your Issue, please we recommend the followance of those requirements:

Here's what to do: if you need help:

* Inform a complete message of your problem, and what's happening.
* Attach your **lumen.log** if it exists.
* Attach images of the reported bugs/glitches.
* Attach also Information about Operational System, Web Server, PHP Versiom, and other infos.
* Explain what you tried and how tried.

Recommendations About your recently opened Issue?
--------------------------------

We really recommend using [Lightshot](https://app.prntscr.com/pt-br/download.html) to take your screenshot. It allows marks, annotations and more.

If you bug it's about a feature from **Chocolatey** or glitch, we **trully** recommend taking a **HAR File**. You can attach a [HAR File following this procedure](https://support.cloudflare.com/hc/en-us/articles/203118044-How-do-I-generate-a-HAR-file-).

A **HAR** file it's a complete log of the network requests of a specific page.

Also we recommend specify some Informations about your **OS** like.:

* Operational System
* Which Web Server are you using (like XAMPP)
* Which version of Web Server Using (eg.: 1.7.3)
* Which version of php & mysql & apache using. (Specify if is possible)
* How much RAM does you have?

Are you creating a Feature Request?
------------------------------

About Feature requests, we recommend you specify completely about what your **Feature Request** is. Tag the type of change that you want to be added.

Changes need to be specified, and you need specify why and what you want to be changed/added.

Be creatively and try explain in the best manner as possible.

Tag the Issue on the Title
-------------------------------

Tagging your Issue will be the best thing you can do for us. We separate issues by tags, and we can work more quickly by knowing from what is the Issue related.

**Those are the available tags:**

* Security
* Angular
* Feature Request
* Server Error
* HabboWEB Glitch
* Translation Mistake
* CSS Mistake/Glitch
* Feature not Working
* Help Wanted
* Can't Install
* Urgent Issue
* Game Bug
* Migration Bug
* Upgrading Bug

**Remember to tag on the title of the Issue [_Issue Type_]**

I updated my Issue.. And now?
---------------------------

You wait.

We have **ful-jobs** and a **life**, we will receive updates by e-mail and soon as possible update your **Issue** by tagging it correctly, assigning to our team and **obviously** answering it.

Then we will support you and help you in what we can.

Also if you Issue it's out of the **params** we will not answer it, or answer it **asking to you fix and correct the main message**

If you dom't know how to use GitHub correctly, check [this](https://guides.github.com/features/issues/) tutorial.

Further information
-------------------

* This **BOT** was originally created by @ajdlinux and @daxtens.
* Edited by @sant0ro for own purposes.
* Contributors of Chocolatey @BobyTS and @s150217754

**Comment by sant0ro** [comment]
Hi @Iluminatos!

Thanks for reporting an Issue to the **Chocolatey Repository**.

We're glad that you're using our software. I need to remember that **this** message was generated automatically.

In order to **gracefully** report your Issue, please we recommend the followance of those requirements:

Here's what to do: if you need help:

* Inform a complete message of your problem, and what's happening.
* Attach your **lumen.log** if it exists.
* Attach images of the reported bugs/glitches.
* Attach also Information about Operational System, Web Server, PHP Versiom, and other infos.
* Explain what you tried and how tried.

Recommendations About your recently opened Issue?
--------------------------------

We really recommend using [Lightshot](https://app.prntscr.com/pt-br/download.html) to take your screenshot. It allows marks, annotations and more.

If you bug it's about a feature from **Chocolatey** or glitch, we **trully** recommend taking a **HAR File**. You can attach a [HAR File following this procedure](https://support.cloudflare.com/hc/en-us/articles/203118044-How-do-I-generate-a-HAR-file-).

A **HAR** file it's a complete log of the network requests of a specific page.

Also we recommend specify some Informations about your **OS** like.:

* Operational System
* Which Web Server are you using (like XAMPP)
* Which version of Web Server Using (eg.: 1.7.3)
* Which version of php & mysql & apache using. (Specify if is possible)
* How much RAM does you have?

Are you creating a Feature Request?
------------------------------

About Feature requests, we recommend you specify completely about what your **Feature Request** is. Tag the type of change that you want to be added.

Changes need to be specified, and you need specify why and what you want to be changed/added.

Be creatively and try explain in the best manner as possible.

Tag the Issue on the Title
-------------------------------

Tagging your Issue will be the best thing you can do for us. We separate issues by tags, and we can work more quickly by knowing from what is the Issue related.

**Those are the available tags:**

* Security
* Angular
* Feature Request
* Server Error
* HabboWEB Glitch
* Translation Mistake
* CSS Mistake/Glitch
* Feature not Working
* Help Wanted
* Can't Install
* Urgent Issue
* Game Bug
* Migration Bug
* Upgrading Bug

**Remember to tag on the title of the Issue [_Issue Type_]**

I updated my Issue.. And now?
---------------------------

You wait.

We have **ful-jobs** and a **life**, we will receive updates by e-mail and soon as possible update your **Issue** by tagging it correctly, assigning to our team and **obviously** answering it.

Then we will support you and help you in what we can.

Also if you Issue it's out of the **params** we will not answer it, or answer it **asking to you fix and correct the main message**

If you dom't know how to use GitHub correctly, check [this](https://guides.github.com/features/issues/) tutorial.

Further information
-------------------

* This **BOT** was originally created by @ajdlinux and @daxtens.
* Edited by @sant0ro for own purposes.
* Contributors of Chocolatey @BobyTS and @s150217754

**Comment by sant0ro** [comment]
Hi @Iluminatos please update your main post with the needed information.

What you're exactly trying to achieve?

You want paypal work? As I explained payments need be configured manually. I will create an tutorial on the future.

**Comment by Iluminatos** [comment]
Will do, 

If you press the "BUY" button i want the user to get sent to the paypal page.

Also another question while im at it: Where do i find to change these texts?
https://gyazo.com/205b25e84acbd74b7d4743e411a6487f

And is there some sort of news images archive?

**Comment by sant0ro** [comment]
Chocolatey only needs your camera username. The Camera username & password you put on Arcturus.

Those texts you can change in ´resources/langs/en/` folder. Just check the file that you want to edit.

The BUY button only will work if you do an advanced configuration on Database. Tutorials will be created.

Any further question?

**Comment by Iluminatos** [comment]
_Thanks.
Oh, if i change in /en does it still show on "Global" on web?
Oh ok, thanks._

**Is there a BETA for espreso?** 
**Is there a script for generating VOUCHER codes** (To enable VIP, Credits etc after a purchase).

**Comment by sant0ro** [comment]
No doesn't exists a script for generating codes yet, you can manually insert it on Database.

You can ask directly for the developers of Espreso @Nenji @dominicusdev @Emeto

**Comment by Iluminatos** [comment]
Oh ok. In: **chocolatey_shop_payment_checkout?**

_Could you anwser the other questions aswell._
would be much appreciated, Santoro.
Thanks :)

**Comment by sant0ro** [comment]
Oh ok. In: chocolatey_shop_payment_checkout? Yes.

I don't understood what you meant with "/en"

The /en folder contains all english language templates.

**Comment by Iluminatos** [comment]
**Wich one is the code?**
https://gyazo.com/a65ab41485b92d291f25f24df4261672

**If i edit a file inside the english folder of translation.**
Will then this effect the website if you have "global" as language? :)

**Comment by sant0ro** [comment]
@Iluminatos doesn't exists a global language. The default language enabled is english.

Other, I won't recommend you try by yourself edit the table. A further tutorial will be created. Be patient :)

**Comment by Iluminatos** [comment]
ooh, ok.
Just did, worked fine! Thank you so much!

Last 2 questions:
My camera settings in Choco & EMU & External_varibales is correct, even tho my camera does not show up in client? Do i have to pay anything for it to work? Have bought a subscription @Arcturus.wf

**Wich one here is the VOUCHER code that you enable in the shop?**
https://gyazo.com/a65ab41485b92d291f25f24df4261672

**Comment by sant0ro** [comment]
My camera settings in Choco & EMU & External_varibales is correct, even tho my camera does not show up in client? Do i have to pay anything for it to work? Have bought a subscription @Arcturus.wf

You need change on `permissions` table, enabling the camera permissions.

Voucher is on the `vouchers` table.

**Comment by Iluminatos** [comment]
OOOH THANKS! 💃

**Comment by sant0ro** [comment]
Glad that you solved it. Any further questions please create a new Issue.

**Comment by Iluminatos** [comment]
**Is there some sort of news images archive?**

**Comment by sant0ro** [comment]
Here: http://forum.ragezone.com/f353/share-web_promos-news-images-932498/
